riverdriver/river_driver_interface.go

```diff
@@ -249,44 +248,46 @@ type JobGetStuckParams struct {
 }
 
 type JobInsertFastParams struct {
-	CreatedAt   *time.Time
-	EncodedArgs []byte
-	Kind        string
-	MaxAttempts int
-	Metadata    []byte
-	Priority    int
-	Queue       string
-	ScheduledAt *time.Time
-	State       rivertype.JobState
-	Tags        []string
-}
-
-type JobInsertUniqueParams struct {
-	*JobInsertFastParams
-	UniqueKey []byte
-}
-
-type JobInsertUniqueResult struct {
+	// Args contains the raw underlying job arguments struct. It has already been
+	// encoded into EncodedArgs, but the original is kept here for to leverage its
+	// struct tags and interfaces, such as for use in unique key generation.
+	Args         rivertype.JobArgs
```

Comment:
Related to discussion from #584, this was added to facilitate tag reflection on the `JobArgs` struct, even though it's otherwise unused (`EncodedArgs` are already encoded at this point).